src/app/tests/suites/certification/Test_TC_SMCO_1_1.yaml

```diff
@@ -0,0 +1,96 @@
+# Copyright (c) 2023 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 4.1.1. [TC-SMCO-1.1] Global Attributes with DUT as Server
+
+PICS:
+    - SMCO.S
+
+config:
+    nodeId: 0x12344321
+    cluster: "Smoke CO Alarm"
+    endpoint: 1
+
+tests:
+    - label: "Commission DUT to TH"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
+    - label: "TH reads the ClusterRevision attribute from the DUT"
+      PICS: SMCO.S.Afffd
+      command: "readAttribute"
+      attribute: "ClusterRevision"
+      response:
+          value: 1
+          constraints:
+              type: int16u
+
+    - label: "TH reads from the DUT the FeatureMap attribute"
+      PICS: SMCO.S.Afffc && SMCO.S.F00
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x1]
+
+    - label: "TH reads from the DUT the FeatureMap attribute"
+      PICS: SMCO.S.Afffc && SMCO.S.F01
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x2]
+
+    # TODO: Date type encoding not defined
+    - label: "TH reads from the DUT the AttributeList attribute"
+      PICS: SMCO.S.Afffb
+      command: "readAttribute"
+      attribute: "AttributeList"
+      response:
+          constraints:
+              type: list
+              contains: [0, 1, 2, 3, 5, 6, 7, 4, 8, 9, 10, 11]
```

Comment:
This does not match the test plan, _and_ the test plan is wrong.

1.  Attributes 1 and 2 are not mandatory, so the test plan is wrong to claim they are.
2. This is claiming the optional attributes need to be in the list.  Those should all be conditioned on the relevant PICS, I would think.

For now I guess this should match the test plan and a test plan issue needs to be raised and linked here.